app/src/main/java/com/asfoundation/wallet/ui/ConfirmationActivity.java

```diff
@@ -78,9 +78,11 @@ private void onTransactionBuilder(TransactionBuilder transactionBuilder) {
     fromAddressText.setText(transactionBuilder.fromAddress());
     toAddressText.setText(transactionBuilder.toAddress());
 
-    valueText.setText(getString(R.string.new_transaction_value, transactionBuilder.amount(),
-        transactionBuilder.symbol()));
-    valueText.setTextColor(ContextCompat.getColor(this, R.color.red));
+    String value = "-" + transactionBuilder.amount()
+        .toString();
+    String symbol = transactionBuilder.symbol();
+    int color = getResources().getColor(R.color.gray_alpha_8a);
+    valueText.setText(BalanceUtils.formatBalance(value, symbol, 30, color));
```

Comment:
The size of the currency doesn't seem to be correct. Please use the value from the resources and not a hard coded one.